utils/helpers.py

```diff
@@ -67,6 +67,44 @@ def ensure_dir(file_path: str):
     return file_path
 
 
+class JobPredictions:
+    def __init__(
+        self,
+        job_name: str,
+        external_id_array: List[str],
+        json_response_array: List[Any],
+        model_name_array: List[str],
+        predictions_probability: List[float],
+    ):
+        self.job_name = job_name
+        self.external_id_array = external_id_array
+        self.json_response_array = json_response_array
+        self.model_name_array = model_name_array
+        self.predictions_probability = predictions_probability
+
+        n_assets = len(external_id_array)
+
+        # assert all lists are compatible
+        same_len = n_assets == len(json_response_array)
```

Comment:
no need to declare `same_len`

Reply:
<img width="487" alt="image" src="https://user-images.githubusercontent.com/38733898/162718357-132997a1-5262-4b31-b786-98483c685a59.png">
the formatting is ugly if i do not declare an intermediate variable